app/assets/stylesheets/views/_colhub.scss

```diff
@@ -0,0 +1,9 @@
+.colhub__announcement {
+  background-color: govuk-colour("light-grey");
+  padding: govuk-spacing(3) 0;
+  margin-bottom: govuk-spacing(4);
+  @include govuk-media-query($from: desktop) {
+    padding: govuk-spacing(6) 0;
```

Comment:
Doesn't have to be blocking as this needs to go out urgently but we might be able to use the Design System responsive spacing mixins here instead of setting the padding and margin separately for mobile and desktop.

Reply:
OK, this is something I wasn't aware existed. Happy to revise it later, if it's fine for this quick release.

Reply:
Cool. It's now captured [here](https://trello.com/c/MlXyrLwz/746-remove-fix-inline-changes-for-cost-of-living-hub-page-s) so resolving this comment.